app/frontend/src/pages/chat/Chat.tsx

```
@@ -169,172 +147,165 @@ const Chat = () => {
 
-                        latestResponse = { ...askResponse, answer: answer, tokens: streamed_tokens };
-                        setIsLoading(false);
-                        setAnswers([...answers, [question, latestResponse, user_tokens]]);
+            for await (const chunk of readNDJSONStream(response.body)) {
+                if (chunk as Chunk) {
+                    switch (chunk.type) {
+                        case "C":
+                            answer += chunk.message as string;
+                            break;
+                        case "I":
+                            const info = chunk.message as ChunkInfo;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)